test/utils/withApp.tsx

```diff
@@ -1,14 +1,24 @@
+import { I18nextProvider } from 'react-i18next'
 import { createMemoryRouter, RouterProvider } from 'react-router-dom'
 
 import { withAppContext } from './withAppContext'
 import { withQueryClient } from './withQueryClient'
 
+import i18n from '../../src/i18n'
 import { ROUTES } from '../../src/routes'
 
 export const withApp = (path: string) => {
   const router = createMemoryRouter(ROUTES, {
     initialEntries: [path],
   })
 
-  return withQueryClient(withAppContext(<RouterProvider router={router} />))
+  i18n.changeLanguage('nl')
```

Comment:
Should we add a test whether each language or at least one language is available?

Reply:
The tests already cover that. This file, `withApp.tsx`, is imported by those tests to use i18n and set the language we are testing on, Dutch in this case. If certain keys in the Dutch translations are missing or the file is completely missing the tests will fail.